packages/remix-react/errors.ts

```diff
@@ -1,15 +1,10 @@
 import type { Router as RemixRouter } from "@remix-run/router";
 import { ErrorResponse } from "@remix-run/router";
 
-import type { AppData } from "./data";
-
-export interface ThrownResponse<
-  Status extends number = number,
-  Data = AppData
-> {
-  status: Status;
+export interface ThrownResponse {
+  status: number;
```

Comment:
This would break usage of the interface where people know that they only throw `400` & `404` for instance